autoload/phpactor/project/bufferMatcher.vim

```diff
@@ -0,0 +1,129 @@
+function! phpactor#project#bufferMatcher#assignFileToProject(filename)
```

Comment:
Rename to `assignProjectToBuffer`?

Reply:
ProjectAssigner instead of BufferMatcher?

Reply:
or just `assigner`? as we have `project` in the namespace?

Reply:
`#phpactor#project#assigner#assignBufferToProject`?